### PR TITLE
[Gui] Remove travis and make changes to use github actions

### PR DESCRIPTION
**Pull request by marianoleonardo**
dojot/dojot#1913

### CODE REVIEW

**Review by mprevide — DISMISSED** [review]
Great job! :+1:

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      uses: actions/checkout@v1
+
+    - name: Build and Push
+      uses: elgohr/Publish-Docker-Github-Action@2.16
+      with:
+        name: ${{ steps.get_owner.outputs.owner }}/gui
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+        # Label with the built image's commit
+        buildoptions: " --label 'commit_sha'='${{ github.sha }}' "
```
Comment (mprevide) [inline .github/workflows/main.yml:59]: Change to `buildoptions: ' --label commit_sha_dojot=${{ github.sha }} '` in this yaml and others you will create in other repositories! :)

Comment (mprevide) [reply .github/workflows/main.yml:59]: I need to fix this in dojot/dojot, dojot/sdk.. and dojot/gui-v2

**Comment by mprevide** [comment]
Sorry, there is one more details, just in guis, that I forget, you need to pass the arg DOJOT_VERSION.
It's just copy from here the idea:  https://github.com/dojot/gui-v2/blob/b9dcc27be4161efe5ed3edb4d4b19360e72c5165/.github/workflows/gui-v2.yml#L77

**Review by mprevide — CHANGES_REQUESTED** [review]
I left some comments

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -46,16 +46,27 @@ jobs:
         export OWNER=$(echo ${{ github.repository }} | cut -f1 -d"/")
         echo Current Owner is $OWNER
         echo ::set-output name=owner::$OWNER
+
     - name: Checkout
       uses: actions/checkout@v1
 
+      - name: Release Version
```
Comment (mprevide) [inline .github/workflows/main.yml:53]: Maybe this indentation will case some error

**Review by mprevide — APPROVED** [review]
Great! :1st_place_medal:

**Review by thiagodpf — APPROVED** [review]

**Pull request merged**